modules/RouterUtils.js

```diff
@@ -1,26 +1,7 @@
-import deprecateObjectProperties from './deprecateObjectProperties'
-
 export function createRouterObject(history, transitionManager) {
   return {
     ...history,
     setRouteLeaveHook: transitionManager.listenBeforeLeavingRoute,
```

Comment:
`delete router.listen`? I'd be okay with that. Maybe we should deprecate it first and see if anybody complains, though.